### PR TITLE
Improve console logging - Closes #4812

### LINKED ISSUE #4812: Console logger should not require bunyan to parse
### Description

Currently console logging is displayed in the JSON format, but it should be displayed in the human readable way.

### Motivation

Console logging is mostly for development, and in the production, only file logging should be enabled.
Therefore, although it might have slight performance degrade on the console logging, it should be better to display the human readable log.

### Acceptance Criteria

- console log should not require piping `npx bunyan`

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #4812 

### How was it solved?

Add formatting before writing to stdout

### How was it tested?

Run the application and observe the log

### CODE REVIEW

**Comment by shuse2** [comment]
@nazarhussain 
I think it's hacky to use the executable from the application. Let's use bunyan to format only if they provide library API

File: framework/src/components/logger/logger.js [hunk 1]
```diff
@@ -26,6 +27,55 @@ const createDirIfNotExist = filePath => {
...
+	50: setColor('red', 'ERROR'),
+	60: setColor('red', 'FATAL'),
+};
+
+class ConsoleLog {
+	// eslint-disable-next-line
+	write(rec) {
+		try {
+			let log = util.format(
+				'%s [%s] %s %s (module=%s)\n',
```
Comment (nazarhussain) [inline framework/src/components/logger/logger.js]:
```suggestion
				'%s %s %s: %s (module=%s)\n',
```

File: framework/src/components/logger/logger.js [hunk 2]
```diff
@@ -26,6 +27,55 @@ const createDirIfNotExist = filePath => {
...
+				new Date(rec.time).toISOString(),
```
Comment (nazarhussain) [inline framework/src/components/logger/logger.js]:
```suggestion
				new Date(rec.time).toLocaleTimeString('en-US', { hour12: false }),
```

Comment (shuse2) [reply framework/src/components/logger/logger.js]: I think it's more closer to the bunyan, but for log, i think it's better to have the date too

Comment (nazarhussain) [reply framework/src/components/logger/logger.js]: For a certain session, it will just be the same date repeating every line.

**Pull request merged**